assets/PROS/JINX/src/JINX.c

```diff
@@ -99,16 +99,16 @@ int readLine(JINX *inStr) {
 
 //Get tokenNumth token of incoming string, and put it in inStr.token
 //Do not pass a null pointer!
-int getToken(JINX *inStr, int tokenNum) {
+int getToken (JINX *inStr, int tokenNum) {
```

Comment:
Functions and typecasts have no space before the parenthesis. Control flow statements (if/while, etc.) do.